index.html

```diff
@@ -177,33 +263,34 @@ <h5 class="connect">LET'S CONNECT</h5>
                 <li><img src="./img/Ellipse.png" alt="CSS"></li>
                 <li><p>CSS</p></li>
                   </ul>
+            </div>
             <ul class="languages">
                 <li><p>Frameworks</p></li>
-                <li><img src="./img/arrow1.png" alt="arrow"></li>
+                <li><img src="./img/arrow2.png" alt="arrow2"></li>
                 </ul>
             <ul class="languages">
                 <li><p>Skills</p></li>
-                <li><img src="./img/arrow1.png" alt="arrow"></li>
+                <li><img src="./img/arrow2.png" alt="arrow2"></li>
                 </ul>
        </div>
-       </div>
      </div>
-
-
-    <section>
-      <div class="form-container">
+<!-- End about myself...................................... -->
+<!-- Start Contact Form ........... -->
+      <section class="form-container">
```

Comment:
- [ ] I think it's better to correct the styling of this section. It's a project requirement.
Your app:
<img width="1278" alt="Screen Shot 2022-08-27 at 12 34 12 AM" src="https://user-images.githubusercontent.com/47673311/187012896-939df24a-2ca7-46f1-8ae8-f96015f64b27.png">

Figma template:
<img width="681" alt="Screen Shot 2022-08-27 at 12 34 45 AM" src="https://user-images.githubusercontent.com/47673311/187012903-ff39c274-04be-4166-80fb-a6957638b95a.png">

Reply:
- [ ] You did a great job here 💯 However, To match your Figma design The elements ("contact me" and the description) of the form should be centered on the page 🙏

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/187036983-a063fcac-fe9f-43f2-a3b9-4126a2c372fb.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/187037020-ffd12818-f687-4cac-90f1-ca10e40cd205.png" width="500" /> |